blocks/identity-block/features/header-account-action/default.jsx

```diff
@@ -48,6 +57,42 @@ const HeaderAccountAction = ({ customFields }) => {
     return null;
   }
 
+  const AccountMenu = () => (
+    <ul className="menu">
+      <PrimaryFont as="li" className="menu-item">Manage Your Account</PrimaryFont>
+      <PrimaryFont as="li" className="menu-item">Log Out</PrimaryFont>
+    </ul>
+  );
+
+  const AccountDropdown = () => (
+    <div className="xpmedia-subs-header">
+      <button type="button" className="xpmedia-subs-header--button" onClick={() => setAccountMenu(!accountMenuToggle)}>
+        <UserIcon fill="#fff" height={accountIconSize} width={accountIconSize} />
+        <PrimaryFont
+          as="span"
+          primaryColor={getThemeStyle(arcSite)['primary-color']}
```

Comment:
`<PrimaryFont />` has a prop named `fontColor` which is the string used in the `getThemeStyle(arcSite)` lookup - which allows you to pass in `fontColor="primary-color'`